products/idn/api/authentication.md

```diff
@@ -304,34 +304,43 @@ An OAuth 2.0 client using the Client Credentials flow must have `CLIENT_CREDENTI
 }
 ```
 
-[Personal Access Tokens](#personal-access-tokens) are implicly granted a `CLIENT_CREDENTIALS` grant type.
+PATs are implicitly granted the `CLIENT_CREDENTIALS` grant type.
 
-The overall authorization flow looks like this:
+This is the overall authorization flow:
 
-1. The client submits an **OAuth 2.0 Token Request** to IdentityNow in the form:
+1. The client first submits an OAuth 2.0 token request to IDN in this form:
 
 ```text
-POST https://{tenant}.api.identitynow.com/oauth/token?grant_type=client_credentials&client_id={client-id}&client_secret={client-secret}
+POST https://{tenant}.api.identitynow.com/oauth/token
 ```
 
```

Comment:
I think it might be helpful to show the json of the body here to show how it could be formatted in a tool like postman instead of only showing the curl command below